src/protocol/chromium-bidi.ts

```diff
@@ -96,6 +106,11 @@ export type Command = (
       // not re-define it. Therefore, it's not part of generated types.
       id: WebDriverBidi.JsUint;
     } & WebDriverBidiPermissions.PermissionsCommand)
+  | ({
+      // id is defined by the main WebDriver BiDi spec and extension specs do
```

Comment:
Can we update the type generator, so that it generates the `id` for all the commands?

Reply:
Would you like to look into it? Note that this file is not generated.